runtime/compiler/runtime/JITClientSession.cpp

```diff
@@ -547,6 +552,22 @@ ClientSessionData::clearCaches()
       }
 
    _wellKnownClasses.clear();
+   setCachesAreCleared(true);
+   }
+
+void
+ClientSessionData::clearCachesLocked(TR_J9VMBase *fe)
+   {
+   // Acquire all required locks to clear client session caches
+   // This should be used when caches need to be cleared
+   // while the client is still sending requests
+   getSequencingMonitor()->enter();
+   writeAcquireClassUnloadRWMutex();
+   OMR::CriticalSection processUnloadedClasses(getROMMapMonitor());
+   TR::ClassTableCriticalSection deleteCHTable(fe);
```

Comment:
Looking at the code, deleting the CHTable means deleting the monitor used here, so we cannot acquire this monitor. Luckily, there is no need to do so.